cypress/e2e/account/selfEditAccountSettings.feature

```diff
@@ -0,0 +1,34 @@
+@regression_test
+@stable_test
+Feature: Account - Edit e-mail in the account setting page on dBC
+
+    As a user I want to navigation to my account's settings page so that I can see and edit my account details (e. g. email).
+
+    Scenario: I see my e-mail is editable as an internal user, including pre conditions
```

Comment:
another point in this scenario which is lacking to me, where we are checking login by actually login with the changed email? right now its just checking success message.